qemu/tests/cfg/hv_check_cpu_utilization.cfg

```
@@ -17,6 +17,9 @@
         set_full_control_cmd = 'WIN_UTILS:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\'
         set_full_control_cmd += 'SecurityHealthService" -ot reg -actn ace -ace "n:Administrators;p:full"'
         service_stop_cmd = sc stop {0} & reg add "HKLM\SYSTEM\CurrentControlSet\Services\{0}" /v Start /d 4 /t REG_DWORD /f
+    i386:
+        Win10:
+            service_stop_cmd = sc stop {0} & reg add "HKLM\SYSTEM\CurrentControlSet\Services\{0}" /v Start /d 4 /t REG_DWORD /f
```

```suggestion
    Win10:
        i386:
            service_stop_cmd = sc stop {0} & reg add "HKLM\SYSTEM\CurrentControlSet\Services\{0}" /v Start /d 4 /t REG_DWORD /f
```
We have to add a similar line for other arch if there is a different cmd for Win OS and this layout also align to the previous filter `Win11:`
Or maybe we could have a try with the combine filter `Win10.i386:`
```suggestion
    Win10.i386:
        service_stop_cmd = sc stop {0} & reg add "HKLM\SYSTEM\CurrentControlSet\Services\{0}" /v Start /d 4 /t REG_DWORD /f
```

---

Thanks for advices, trying to test all winodws guests with the same service_stop_cmd.
Actually, fixed bug only hit on win10-32 before~

If all test can be passed with new service stop method, will use the new method to all guests, otherwise, will use the second suggestion.

Thanks~
Peixiu

---

> Thanks for advices, trying to test all winodws guests with the same service_stop_cmd. Actually, fixed bug only hit on win10-32 before~
> 
> If all test can be passed with new service stop method, will use the new method to all guests, otherwise, will use the second suggestion.
> 
> Thanks~ Peixiu

Okay, looks good to me